deployments/container/Dockerfile

```diff
@@ -34,14 +34,33 @@ ARG GOLANG_VERSION=x.x.x
 ARG BUILDARCH
 ARG TARGETARCH
 
+# Install dependencies for Go build. Do not clear apt cache (does not
+# leak into prod stage).
 RUN apt-get update && \
     apt-get install -y \
         wget \
         make \
         git \
         gcc-aarch64-linux-gnu \
-        gcc && \
-    rm -rf /var/lib/apt/lists/*
+        gcc
+
+# Install dependencies for `bash-static` build.
+RUN apt-get install -y gpg curl autoconf
+
+# Build static bash binary (against musl).
+WORKDIR /bashbuild
+RUN git clone https://github.com/robxu9/bash-static/
```

Comment:
Did we want a specific tag here?

Reply:
Ah ... wait ... I just saw it below. Could we do a shallow clone at a specific commit?

Reply:
(answering my own quesiton: It seems as if the releases are VERY out of date).

Reply:
> releases are VERY out of date

and: yes -- I vetted the repo a bit, and I think it's generally in a serious / ok state. Except for the releases section :). 

Most importantly, the latest proper patch release for bash is [bash-5.2.37.tar.gz](https://ftp.gnu.org/gnu/bash/bash-5.2.37.tar.gz) and the project takes that into account via https://github.com/robxu9/bash-static/blob/021f5f29f665c92ca16a369d9f27e288c3aed0c6/version-52.sh#L4.